cmd/operator-sdk/new/cmd.go

```diff
@@ -117,35 +112,23 @@ generates a default directory layout based on the input <project-name>.
 		"Do not validate the resulting project's structure and dependencies. (Only used for --type go)")
 	newCmd.Flags().BoolVar(&generatePlaybook, "generate-playbook", false,
 		"Generate a playbook skeleton. (Only used for --type ansible)")
-	newCmd.Flags().StringVar(&helmChartRef, "helm-chart", "",
-		"Initialize helm operator with existing helm chart (<URL>, <repo>/<name>, or local path)")
-	newCmd.Flags().StringVar(&helmChartVersion, "helm-chart-version", "",
-		"Specific version of the helm chart (default is latest version)")
-	newCmd.Flags().StringVar(&helmChartRepo, "helm-chart-repo", "",
-		"Chart repository URL for the requested helm chart")
-	newCmd.Flags().StringVar(&crdVersion, "crd-version", gencrd.DefaultCRDVersion,
-		"CRD version to generate (Only used for --type=ansible|helm)")
+
+	// Initialize flagSet struct with common flags
+	fs.AddTo(newCmd.Flags())
 
 	return newCmd
 }
 
 var (
-	apiVersion       string
-	kind             string
-	operatorType     string
+	fs               apiflags.APIFlags
```

Comment:
IMO, fs is not intuitive. it remembers me file system a not APIFlags. 
Could we rename it for apiflags?

Reply:
@bharathi-tenneti agreed rename `fs` to `flags` if it doesn't collide with something else. I think I would do one of the following:

1) rename `fs` to `flags`
OR
2. a. alias the apiflags import: `theapiflags "github.com/operator-framework/operator-sdk/internal/flags/apiflags"`
2. b. rename `fs` to `apiflags`. Not sure if that will cause more confusion.
OR
3. rename `fs` to `apiflag` singular.

Either 1 or 3 probably is the least confusing option. I'll be okay with either one.